newsfragments/263.bugfix.md

```diff
@@ -0,0 +1,4 @@
+Struct initialization would often fail with a type error since the data structure 
+responsible for managing struct fields does not enforce order.
+
+Order is now accounted for separately.
```

Comment:
We don't need to put a release notes entry for this bugfix since the bug didn't make it into a release